cluster/aws/util.sh

```diff
@@ -270,6 +270,23 @@ function upload-server-tars() {
     # We default to us-east-1 because that's the canonical region for S3,
     # and then the bucket is most-simply named (s3.amazonaws.com)
     aws s3 mb "s3://${AWS_S3_BUCKET}" --region ${AWS_S3_REGION}
+
+    local attempt=0
+    while true; do
+      if ! aws s3 ls "s3://${AWS_S3_BUCKET}" > /dev/null 2>&1; then
+        if (( attempt > 5 )); then
+          echo
+          echo -e "Unable to confirm bucket creation." >&2
```

Comment:
Ah. Fair point. I was copying from the minion check. Which looks like it was copied from the salt-master check. Which actually uses `-e` for a reason.